reference/recency-surveillance.v1.json

```diff
@@ -140,13 +164,21 @@
                         "type": "string"
                       },
                       "health_unit_testing_entry_point_other": {
-                        "type": "string"
+                        "type": "string",
+                        "default": "text",
+                        "enum": [
```

Comment:
@ODORA0 This cannot be an enum since its text based on the value of health_unit_testing_entry_point